authlib/integrations/base_client/async_openid.py

```diff
@@ -16,7 +16,7 @@ async def fetch_jwk_set(self, force=False):
             raise RuntimeError('Missing "jwks_uri" in metadata')
 
         async with self.client_cls(**self.client_kwargs) as client:
-            resp = await client.request('GET', uri, withhold_token=True)
+            resp = await client.request('GET', uri, withhold_token=True, **self.client_kwargs)
```

Comment:
❓ Are there no tests to update for this case?

Reply:
There are no tests for `fetch-jwk` or for `load_server_metadata` for neither the sync app or the async app. I added this commit to make sure the async app works the same as the sync app. I did add regression tests for the sync app though as it's important for us.